Makefile

```diff
@@ -29,11 +29,18 @@ VDDK_LIBS:= $(GVDDK)/vmware-vix-disklib-distrib/lib64
 # The binary to build (just the basename).
 PLUGIN_BIN ?= $(wildcard velero-*)
 DATAMGR_BIN ?= $(wildcard data-manager-*)
+BACKUPDRIVER_BIN ?= backupdriver
+
+# The input path (just the basename).
+PLUGIN_INP ?= $(wildcard velero-*)
+DATAMGR_INP ?= $(wildcard data-manager-*)
+BACKUPDRIVER_INP ?= cmd/backupdriver
```

Comment:
You don't have to introduce these *_INP. You can do as below
```
PLUGIN_BIN ?= velero-plugin-for-vsphere
DATAMGR_BIN ?= data-manager-for-plugin
BACKUPDRIVER_BIN ?= backupdriver
```
And move velero-plugin-for-vsphere.go and data-manager-for-plugin.go to cmd/ folder. Then change the last line of `hack/build.sh` script from `${PKG}/${BIN}` to `${PKG}/cmd/${BIN}`